### ISSUE TITLE
Better error messages parsing json

### ISSUE THREAD

**Issue by melizeche** [mrbenshef/Boop-GTK#148]
It would be nice to have better error message formatting JSON, sometimes I paste 30 lines of JSON and if everything isn't perfect it just says `ERROR: Invalid JSON` I think a great improvement would be to have some extra information like https://jsonlint.com/
![image](https://user-images.githubusercontent.com/484773/96196862-de702f00-0f26-11eb-8a0c-fa92401c7381.png)

PS: I really like this project, awesome work!

**Comment by zoeyfyi** [comment]
I agree, do you want to open an issue upstream (https://github.com/IvanMathy/Boop/)? I know @IvanMathy was [keen to keep `JSON.parse` because its native](https://github.com/IvanMathy/Boop/issues/167#issuecomment-694266798), and providing better error messages would require a custom parser, but IMO the usability improvements from better error messages seems worth it.

**Comment by melizeche** [comment]
@mrbenshef I wasn't aware of the existence of Boop for mac, I thought this was the "main" project. Thanks for pointing me in the right direction I opened an issue there --> https://github.com/IvanMathy/Boop/issues/202
